src/StepperFirmata2.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (C) 2006-2008 Hans-Christoph Steiner.  All rights reserved.
+  Copyright (C) 2010-2011 Paul Stoffregen.  All rights reserved.
+  Copyright (C) 2009 Shigeru Kobayashi.  All rights reserved.
+  Copyright (C) 2013 Norbert Truchsess. All rights reserved.
+  Copyright (C) 2009-2016 Jeff Hoefs.  All rights reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  See file LICENSE.txt for further informations on licensing terms.
+*/
+
+#ifndef StepperFirmata2_h
+#define StepperFirmata2_h
+
+#include <ConfigurableFirmata.h>
+#include "utility/AccelStepper.h"
+#include "utility/MultiStepper.h"
+#include "FirmataFeature.h"
+
+#define MAX_STEPPERS 10 // arbitrary value... may need to adjust
+#define MAX_GROUPS 5 // arbitrary value... may need to adjust
```

Comment:
defines are global across all files and therefore should be named somewhat specifically. Please rename to `MAX_STEPPER_GROUPS`.